gaffer-as-a-service/gaas-rest/deploy/openshift-prometheus.yaml

```diff
@@ -0,0 +1,121 @@
+kind: Template
+apiVersion: v1
+items:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    annotations:
+      openshift.io/generated-by: OpenShiftNewApp
+    labels:
+      app: prometheus
+      app.kubernetes.io/component: prometheus
+      app.kubernetes.io/instance: prometheus
+    name: prometheus
+  spec:
+    ports:
+    - name: 9090-tcp
+      port: 9090
+      protocol: TCP
+      targetPort: 9090
+    selector:
+      deployment: prometheus
+    sessionAffinity: None
+    type: ClusterIP
+  status:
+    loadBalancer: {}
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    annotations:
+      deployment.kubernetes.io/revision: "13"
+      image.openshift.io/triggers: '[{"from":{"kind":"ImageStreamTag","name":"prometheus:latest"},"fieldPath":"spec.template.spec.containers[?(@.name==\"prometheus\")].image"}]'
+      openshift.io/generated-by: OpenShiftNewApp
+    generation: 29
+    labels:
+      app: prometheus
+      app.kubernetes.io/component: prometheus
+      app.kubernetes.io/instance: prometheus
+    name: prometheus
+    resourceVersion: "30495790"
+    selfLink: /apis/apps/v1/namespaces/kai-dev/deployments/prometheus
+    uid: 7fbe5440-dd6a-4100-a18e-de614b4363e0
+  spec:
+    progressDeadlineSeconds: 600
+    replicas: 1
+    revisionHistoryLimit: 10
+    selector:
+      matchLabels:
+        deployment: prometheus
+    strategy:
+      rollingUpdate:
+        maxSurge: 25%
+        maxUnavailable: 25%
+      type: RollingUpdate
+    template:
+      metadata:
+        annotations:
+          openshift.io/generated-by: OpenShiftNewApp
+        creationTimestamp: null
+        labels:
+          deployment: prometheus
+      spec:
+        containers:
+        - image: prom/prometheus@sha256:3e6f1ebc047ec30971c92e86aa3b879b07d39327a7744cce8aa0b1076d4c94d6
```

Comment:
Tying it to a particular image. Don't know if you want to or not?

Reply:
done